integreat_cms/cms/templates/linkcheck/link_list_row.html

```diff
@@ -5,122 +5,108 @@
 {% load widget_tweaks %}
 {% load linkcheck_filters %}
 {% object_translation_has_view_perm request.user url.region_links.0.content_object as show_source_link %}
-<tbody class="hover:bg-gray-100">
```

Comment:
I liked the previous solution with the `tbody` because it caused the area of the form to also change the hovered background when the `tr` is hovered, which made it more obvious that the two rows actually belong together...

Reply:
True. Please check now